pkg/build/webhook/webhook.go

```diff
@@ -3,9 +3,12 @@ package webhook
 import (
 	"crypto/hmac"
 	"errors"
+	"net/http"
 	"strings"
 
-	"github.com/openshift/origin/pkg/build/api"
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
```

Comment:
of course not. only if i find myself writing something that might collide, but it's good to know. thanks